exporters/otlp/test/otlp_metrics_serialization_test.cc

```diff
@@ -43,6 +43,33 @@ static metrics_sdk::MetricData CreateSumAggregationData()
   return data;
 }
 
+static metrics_sdk::MetricData CreateUpDownCounterAggregationData()
+{
+  metrics_sdk::MetricData data;
+  data.start_ts = opentelemetry::common::SystemTimestamp(std::chrono::system_clock::now());
+  metrics_sdk::InstrumentDescriptor inst_desc = {"UpDownCounter", "Robot Pose Y", "Meter",
+                                                 metrics_sdk::InstrumentType::kUpDownCounter,
+                                                 metrics_sdk::InstrumentValueType::kDouble};
+  metrics_sdk::SumPointData s_data_1, s_data_2;
+  s_data_2.value_ = 1.35;
+  s_data_2.value_ = 1.37;
+
+  data.aggregation_temporality = metrics_sdk::AggregationTemporality::kCumulative;
+  data.end_ts = opentelemetry::common::SystemTimestamp(std::chrono::system_clock::now());
+  data.instrument_descriptor = inst_desc;
+  metrics_sdk::PointDataAttributes point_data_attr_1, point_data_attr_2;
+  point_data_attr_1.attributes = {{"environment_id", "DEV-AYS"}};
+  point_data_attr_1.point_data = s_data_1;
+
+  point_data_attr_2.attributes = {{"robot_id", "DEV-AYS-03-02"}};
+  point_data_attr_2.point_data = s_data_1;
```

Comment:
This should be:
`point_data_attr_2.point_data = s_data_2;`

Again, SumDataAggregation, will fix that later :)